package/com.unity.formats.usd/Tests/Common/BaseFixture.cs

```diff
@@ -30,7 +30,8 @@ public abstract class BaseFixture
         protected string ArtifactsDirectoryFullPath => Path.Combine(Application.dataPath, ArtifactsDirectoryName);
         protected string ArtifactsDirectoryRelativePath => Path.Combine("Assets", ArtifactsDirectoryName);
         protected string TestAssetDirectoryName => TestAssetData.Directory.FolderName;
-        protected string TestUsdAssetDirectoryRelativePath => Path.Combine("Packages", "com.unity.formats.usd", "Tests", "Common", "Data", TestAssetDirectoryName);
+        protected string TestPackageName => Directory.Exists(Path.Combine("Packages", "com.unity.formats.usd", "Tests")) ? "com.unity.formats.usd" : "com.unity.formats.usd.tests";
+        protected string TestUsdAssetDirectoryRelativePath => Path.Combine("Packages", TestPackageName, "Tests", "Common", "Data", TestAssetDirectoryName);
```

Comment:
Is this potentially why other tests use the GUID for the test assets rather than the path? I don't mind this approach, but if that's the way we work around it in other tests it would be good to be consistent here.

Reply:
Great, yeah it would be nice to remove the hardcoded paths. This is ok for now though :)